media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
```

Comment:
We don't really have a policy on the default box-model used in bedrock per-say. I know many dev's do use this as modern default today, but I do wonder if we should maintain some sort of consistency across pages on bedrock. We do need to support older browsers on certain pages, and it can be a bit of a mind-shift swapping from one box modal to another depending upon the page.

Any thoughts on this on @craigcook or @jpetto ?